tests/Microsoft.Identity.Web.Test/TheoryDataBase.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Xunit;
+
+namespace Microsoft.Identity.Web.Test
+{
+    public class TheoryDataBase : TheoryData
```

Comment:
The names of the tests ends up being the ToString on these. You can see this in the Test Exporer.

1. I'd make test id required
2. Incorporate the test id into the ToString method.

Reply:
done 🫡